LibraryWebApp/Views/Author/Index.cshtml

```diff
@@ -23,13 +23,13 @@
             <thead>
                 <tr>
                     <th>
-                        <label asp-for="@modelTemplate.Id"/>
+                        <label asp-for="@modelTemplate.Id" ></label>
```

Comment:
if you have an empty tag in html you can close it right away. <label asp-for="@modelTemplate.Id" />